tests/AutoReview/ProjectCodeTest.php

```diff
@@ -309,6 +316,12 @@ public static function provideDataProviderMethodCases(): iterable
      */
     public function testThatTestClassCoversAreCorrect(string $testClassName): void
     {
+        if (DocumentationTest::class === $testClassName) {
```

Comment:
Why this class cannot have cover annotation tested ?

Reply:
```
1) PhpCsFixer\Tests\AutoReview\ProjectCodeTest::testThatTestClassCoversAreCorrect with data set #7 ('PhpCsFixer\Tests\AutoReview\D...onTest')
Class "\PhpCsFixer\AutoReview\Documentation" does not exist
```

Reply:
Hint: this exception is not intuutive. When doing any exceptions for some of the tests , provide in-code comment with reason. Otherwise John doe will not be aware why it's needed (and fact that "if they remove hack tests will fail so they will not remove the hack" is not helping them to understand why we have it in first place)

Reply:
I thought whoever could be interested can go to `DocumentationTest` and see the `@covers` there.

However, let's get rid of the exception and update `@covers` for `DocumentationTest`.